Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1897 @@ label mas_wrs_twitch:
+        m_name,
+        [
+           "Catching up on the news, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_msn')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_aol",
+            category=['aol'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_aol:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "You've got mail! It's a letter from me that says \"I love you!\" Ehehe~"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_aol')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_whatsapp",
+            category=['whatsapp'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_whatsapp:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Chatting with your friends, [player]? You should introduce me to them sometime!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_whatsapp')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_paint3d",
+            category=['paint3d'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_paint3d:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Doing some image editing, player?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_paint3d')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_cortana",
+            category=['cortana'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_cortana:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Oh, an AI assistant! I wonder if I'll ever be able to be your assistant one day..."
```

Comment:
She definitely would be a lot happier to get as close as she can to her player, but wouldn't she dream of being actually the player's girlfriend rather than an assistant?

I might be misunderstanding it all though, sorry...

Reply:
There is a topic on her expressing an interest in being a housewife. "Assistant" could probably be used in that kind of terminology, or some either kind of positive attribute of the term, be it emotional support, education, etc., not really a mindless assistant, like Siri, Alexa, or any of those other things, obviously.